packages/hardhat-core/src/internal/hardhat-network/jsonrpc/client.ts

```diff
@@ -37,6 +37,15 @@ export class JsonRpcClient {
     return this._networkId;
   }
 
+  public async getDebugTraceTransaction(transactionHash: Buffer): Promise<any> {
```

Comment:
This is only used in a test, right? It surprised me when I first saw it, as we trace remote txs locally.

Reply:
Yes, this is only used by the test script.